src/shared/interfaces/Event.ts

```diff
@@ -0,0 +1,19 @@
+export interface Event {
```

Comment:
id is missing and which of the properties can be optional?

Reply:
1. Added id
2. Added winners and phone for contact to be optional